client/engine/engine.go

```diff
@@ -149,7 +149,8 @@ func (e *Engine) Run() {
 
 		// Handle errors
 		if err != nil {
-			e.logger.Panic(fmt.Errorf("%s, error in run loop: %w", e.store.GetAddress(), err))
+			e.logger.Err(fmt.Errorf("%s, error in run loop: %w", e.store.GetAddress(), err))
+			e.logger.Panic()
```

Comment:
Should this be collapsed into a single line? From the zerolog docs:

"Panic starts a new message with panic level. The panic() function
is called by the Msg method, which stops the ordinary flow of a goroutine.

You must call Msg on the returned event in order to send the event."